source/tutorials/kubernetes.rst

```diff
@@ -3,354 +3,716 @@
-workloads or  multi-tenant scenarios. Kata Containers can be
-allocated on a per-pod basis, so you can mix and match both on the same host
-to suit your needs.
+  * `What is CRI+O?`_
+
+  * `What is containerd?`_
+
+  * `What is Docker?`_
+
+* `What is Kata Containers?`_
 
 Prerequisites
 *************
 
 This tutorial assumes you have already installed |CL|. For detailed
-instructions on installing |CL| on a bare metal system, follow the
-:ref:`bare metal installation tutorial<bare-metal-install-desktop>`.
-Learn about the benefits of having an up-to-date system for cloud
-orchestration on the :ref:`swupd-guide` page.
+instructions on installing |CL| on a bare metal system, follow the :ref:`bare
+metal installation tutorial<bare-metal-install-desktop>`.
+
+#. Before you continue, update your |CL| installation with the following command:
+
+   .. code-block:: bash
 
-Before you install any new packages, update |CL| with the following command:
+      sudo swupd update
 
-.. code-block:: bash
+   Learn about the benefits of having an up-to-date system for cloud
+   orchestration on the :ref:`swupd-guide` page.
 
-   sudo swupd update
+#. Kubernetes, a set of supported :abbr:`CRI (Container Runtime Interface)`
+   runtimes, :abbr:`CNI (Container Network Interface)` and `cloud-native-setup
+   scripts`_ are included in the `cloud-native-basic`_ bundle. Install the
+   cloud-native-basic bundle to get these components:
 
-Install Kubernetes and CRI runtimes
-***********************************
+   .. code-block:: bash
 
-Kubernetes, a set of supported :abbr:`CRI (Container Runtime Interface)`
-runtimes, and networking plugins, are included in the `cloud-native-basic`_
-bundle.
+      sudo swupd bundle-add cloud-native-basic
 
-To install this framework, enter the following command:
 
-.. code-block:: bash
+Set up Kubernetes automatically
+*******************************
 
-   sudo swupd bundle-add cloud-native-basic
+|CL| provides `cloud-native-setup scripts`_ to automate system setup and
+Kubernetes cluster initialization which allows you to get a cluster up and
+running quickly.
 
 .. note::
 
-   For more on networking plugins, see `Install pod network add-on`_.
+   By default, the scripts will update |CL| to the latest version, set up the
+   system as a Kubernetes master-node with **canal for container networking**
+   and **crio for container runtime**, and taint the master node to allow
+   workloads to run on it. Kata is installed as an optional alternative
+   runtime. The script can be configured to use other CNI's and CRI's by
+   following the directions on the `README
+   <https://github.com/clearlinux/cloud-native-setup/blob/master/clr-k8s-examples/README.md>`_.
+   
+   See `What is a Container Network Interface (CNI)?`_ and `What is a
+   Container Runtime Interface (CRI)?`_ for more information.
+
+.. important::
+
+   If network proxy settings are required for Internet connectivity, configure
```

Comment:
It might be worth stating that the script looks at /etc/environment and /etc/profile.d/proxy.sh for proxy information.   I've seen guides state to put proxy info in .bashrc  or /etc/bash.bashrc and the script doesn't check there.

Reply:
👍  Good call. I suggest making this a note.

Reply:
I added this information.